### DISCUSSION TITLE
musikcube 3.0.0

### DISCUSSION THREAD

**Discussion by clangen** [clangen/musikcube#601] [General]
## musikcube 3.0.0 has been released!

Development of this incarnation of musikcube started nearly [7 years ago to to this day](https://github.com/clangen/musikcube/commit/a6b588fcc912fa8081393d2a63d45cf5feb4e12b). In the meantime has seen slow, but (relatively) steady iteration. It began life as a toy, but grew into a powerful, fully-featured desktop application, server, and mobile client.

If you're curious about previous versions of musikcube, and why this particular release is `3.0.0` please [read this short doc on our wiki](https://github.com/clangen/musikcube/wiki/major-release-versions).

### High-level differences between the original musikcube, and musikcube 3 include:
* A completely cross platform terminal-based user interface, currently supporting Windows, macOS, Linux, FreeBSD and Haiku OS
* Redesigned plugin API that allows for audio decoders, encoders, playback and play-queue manipulation, visualizers, data streams, data indexers, digital signal processing and more
* Multi-architecture support, including (but not limited to) x86, amd64, arm64 and armv6
* Headless operation on Unix-based platforms via daemon
* Remote access: stream audio from musikcube instances running on other computers
* Support for ReplayGain during playback
* Full Unicode support across all platforms
* Regex searching and filtering

<hr /><em>This discussion was created from the release <a href='https://github.com/clangen/musikcube/releases/tag/3.0.0'>musikcube 3.0.0</a>.</em>

**Comment by stratacast** [comment]
Thank you for the release! musikcube 3.0.0 finally landed in OpenBSD ports and I can report, it's nice!

**Comment by KenVsChun-Li** [comment]
This is a great tools!  So nice!!!
 Can musikcube  be started as a system service?

**Comment by spstrongco** [comment]
I have really enjoyed MC for a while. I decided to update the 3.0. The .deb file is "not a Debian format archive". Thanks for your help

**Comment by ChaplinTokyo** [comment]
I love Musikcube!  I was looking for a terminal based music player after finding cmus would not work on Haiku, and stumbled on Musikcube on HaikuDepot.  Since this version 3.0.0 is cross-platform, I can use it not only on Haiku, but also on my GhostBSD and FerenOS installed PCs too.  Sweet!  
One question.  I've been trying to discover the Meta key to use in Haiku, and so far have not been able to find it (I thought perhaps that ALT was the Meta key and that it was conflicting with Haiku's use of the ALT as it's keyboard shortcut prefix, and tried switching the ALT and CTRL keys around like in Windows/Linux, but even after doing that, ALT+n did nothing).  What is the Meta key for Musikcube for Haiku please?
